app/src/main-process/menu/build-default-menu.ts

```diff
@@ -229,6 +229,20 @@ export function buildDefaultMenu({
         accelerator: 'CmdOrCtrl+-',
         click: zoom(ZoomDirection.Out),
       },
+      {
+        label: __DARWIN__
+          ? 'Contract Active Resizable'
+          : 'Contract active resizable',
+        accelerator: 'CmdOrCtrl+Shift+-',
+        click: emit('decrease-active-resizable-width'),
+      },
+      {
+        label: __DARWIN__
+          ? 'Expand Active Resizable'
+          : 'Expand active resizable',
+        accelerator: 'CmdOrCtrl+Shift+=',
+        click: emit('increase-active-resizable-width'),
```

Comment:
Again totally open to naming ideas.. I used to have "Increase active resizable width" and "Increase active resizable pane"... but those were both really long. 

Other thoughts. "Expand Active Pane" seems more relatable but maybe vague?